reference/commands/install.rst

```diff
@@ -130,7 +130,8 @@ There are 3 built-in deployers:
   dependencies.
 - ``runtime_deploy`` deploys all the shared libraries and the executables of the
   dependencies (like ``.so``, ``.dll``, or ``.dylib`` files) into a flat directory
-  structure. The symbolic links of libraries are copied by default and they can be managed through the boolean
+  structure, preserving subdirectories as-is.
```

Comment:
I think it would be great to link the reference for each deployer here

Reply:
@czoido I reduced the install to only point to deployers instead: [835e2b9](https://github.com/conan-io/docs/pull/4205/commits/835e2b9dbd92d06e5c3d5be693f6301722aba58b)

I believe the information is duplicated so far, we could centralize all them in the deployers page only. WDYT?

Reply:
For me, having a quick summary of what the deployer does in the install reference was ok, then you can get more details if you want... but I'm fine with this too.

Reply:
Done, the summary is back: [36e5629](https://github.com/conan-io/docs/pull/4205/commits/36e56297f28174f6e69c44d350f5ffaf7a3fdf82)

I reduced a little the runtime_deploy explanation, as we have a more complete information on the deploy page.